### DISCUSSION TITLE
designing-wan-availability/

### DISCUSSION THREAD

**Discussion by giscus** [jvanderaa/jvanderaa.github.io#53] [Announcements]
# designing-wan-availability/

One of my hot topics in my past that I haven’t seen written about often is the calculation of WAN availability and what the design is built for. There is often the number of 9’s whether that is 5 9s or 3 9s or otherwise, where do you start? Well, in the past the post by EventHelix.com outlines system reliability. It talks about designing systems in parallel and in serial. What does that mean.

https://josh-v.com/designing-wan-availability/

**Comment by minicz** [comment]
Thank you Josh! Very interesting article. For long time I didn´t hear about network availability calculation.

Here, you only considered hardware MTBF. To complicate a little, it´s important to consider software and human configuration errors MTBF, too.

**Reply by jvanderaa** [reply]
Agree, there are some assumptions on just the equipment side in that. The software and human side have some mitigations possible with the proper lab testing, and that will be dependent on the environment for sure. That is worth calculating some in and may be a use case for adding some budget to get a lab.
